### DISCUSSION TITLE
Windows 11 taskbar thumbnails do not display title of window in the preview

### DISCUSSION THREAD

**Discussion by brook-ly** [valinet/ExplorerPatcher#564] [Q&A] [answered]
After resetting EP settings to default, I've found myself stuck with this taskbar bug. Uninstalling doesn't solve the problem. Is there any way to fix it?

![image](https://user-images.githubusercontent.com/60946443/146012875-29d848a6-8b9e-456e-aaa1-8275b392d09f.png)

**Comment by valinet** [comment, marked as answer]
What’s the “bug”?

**Reply by brook-ly** [reply]
![image](https://user-images.githubusercontent.com/60946443/146015129-30b9fb1a-16ad-474e-9c81-7b9028b895a3.png)
When hovering the mouse on top of an application the preview window should show the app icon as well as the title. In my case, I only get the preview picture and only get the title after hovering my mouse on top of it.

**Reply by valinet** [reply]
I don’t know. It’s not related to EP, I have seen it work both ways, I don’t remember when and how. Reboot the computer and see if it fixes itself. You could also try asking Microsoft about it, they presumably know more about their software than I do.

**Reply by brook-ly** [reply]
Rebooting doesn't fix it, unfortunately, and as far as this being related to EP this behaviour came directly after resetting its settings to default.

**Reply by Antonomasia3rd** [reply]
On the ExplorerPatcher's Properties, see "Combine taskbar icons on primary taskbar" on "Taskbar" tab, set it to "Always combine" and Restart Explorer
See if it helps! :)
![image](https://user-images.githubusercontent.com/89201774/146020999-091204e0-fdbb-43ac-9306-662c67bfb8e5.png)

**Reply by brook-ly** [reply]
> On the ExplorerPatcher's Properties, see "Combine taskbar icons on primary taskbar" on "Taskbar" tab, set it to "Always combine" and Restart Explorer See if it helps! :) ![image](https://user-images.githubusercontent.com/89201774/146020999-091204e0-fdbb-43ac-9306-662c67bfb8e5.png)

THANK YOU! It worked, navigating between multiple instances of an app with that 'bug' was driving me crazy

**Reply by valinet** [reply]
@Yahiya-Moukhlis Yeah, so it wasn't EP per se. Those 2 settings are the legacy settings that worked to set the taskbar to combine items or display labels since Windows 7. It's interesting they still have an effect on the Windows 11 taskbar as well. This likely suggests that they have partly implemented the uncombined feature in the new taskbar as well, but never bothered to finish and actually ship it. That's interesting, well, I am sure it will eventually ship in one form or another with the new taskbar as well, they probably did not have time to do it.
@abbyurtia Just curious, why is your EP settings window all pink?

**Reply by valinet** [reply]
Anyway, personally I haven't noticed that, it's interesting to know what causes it. Generally, idk much about using Windows 11 with its built-in UI, since July when I installed this, I did my best to revive the old taskbar and rolled with that ever since. I never used the new built-in UI. Also, EP is kind of envisioned for that, I don't quite see what's the use case for using it without enabling the Windows 10 taskbar...

**Reply by Antonomasia3rd** [reply]
> @abbyurtia Just curious, why is your EP settings window all pink?

It follows the "Show accent color on title bars and window borders" on Settings > Personalization > Colors. I thought this is normal (no?)
![image](https://user-images.githubusercontent.com/89201774/146097388-85904dd2-805c-4d99-97a0-ab9f7fb90cad.png)

**Reply by Antonomasia3rd** [reply]
Here is some comparison (in case you need it)
![image](https://user-images.githubusercontent.com/89201774/146097684-391b0a7f-4bd3-494c-b8fc-144671c3b074.png)
![image](https://user-images.githubusercontent.com/89201774/146097696-7365f90c-bbaa-4edc-9940-998bf3fef3f5.png)

**Reply by valinet** [reply]
Yeah, idk. Probably it is. I request Windows to draw the background using the Mica brush. But yeah, what it does depends on what settings the system has. It's interesting. Idk how to reproduce your "setup" so to say. If I enable "Show accent color on title bars and window borders", it still draws the Mica brush over the window on my system, only the border appears blue, as that's the accent color I have set up.

![image](https://user-images.githubusercontent.com/6503598/146116304-14bb531f-16c1-44b4-9bc7-b04533423666.png)

Yeah, whatever, interesting indeed...

**Reply by Antonomasia3rd** [reply]
Oh! The mica brush
Windows 11 breaks that feature in the Latest Dev Build :(

**Reply by valinet** [reply]
Okay =)))))) Nice. Nice. Nice.

**Reply by valinet** [reply]
@abbyurtia And this is exactly the reason why fully supporting Windows Insider Preview builds is a giant paint in the ass. It's too volatile, and this project uses TONS of undocumented things. Also, you say Mica is broken in that build, so it also does not work for the Settings app, for example? if it works there but not in EP, it's likely because the attribute number for Mica that gets passed to `DwmSetWindowAttribute` may have changed. It is1029 in build 22000, maybe now it's 1030 or somewhere around there, they have a habit, unfortunately, of messing with these values however they please, as if there aren't 2 million numbers to choose from anyway. it happened before with attribute 19 (prior to Windows 10 2004) or 20 (after Windows 10 2004) which controls whether `dwm` should display a dark title bar for a window. Yeah, overnight, they figured 19 wasn't cool anymore and made it a 20. Unfortunately, almost all the cool stuff nowadays is undocumented when it comes to Microsoft Windows stuff.

**Reply by Antonomasia3rd** [reply]
Mica effect still works on UWP Apps yeaahh (Settings, for example) but not on some programs (including EP)

**Reply by valinet** [reply]
Yeah, then it means the attribute number likely changed. Typical Microsoft.
